src/test/java/streams/part1/exercise/Exercise2.java

```diff
@@ -48,7 +61,14 @@ public void findEmployeeWithMaximumDurationAtOnePosition() {
     public void calcTotalSalaryWithCoefficientWorkExperience() {
         List<Employee> employees = Example1.getEmployees();
 
-        Double expected = null;
+        int baseSalary = 75_000;
+
+        Double expected = employees.stream()
+                                   .map(employee -> employee.getJobHistory().stream()
+                                                            .reduce((first, second) -> second).orElseThrow(IllegalStateException::new))
```

Comment:
use of streams is redundant here

Reply:
list.get(list.size() - 1) - provides u last element (if the list is not empty, ofc)